pages/[id]/intro.js

```diff
@@ -160,7 +160,11 @@ export default ({ id, hackathons }) => {
   };
 
   return (
-    <Page currentPath={id} hackathons={hackathons} includeNavbar>
+    <Page
```

Comment:
is there a way to add this `Page` to all the pages using a wrapper or container or something? If we end up having to change the props for this, that's a lot of places we're gonna need to change

Reply:
🤔 good idea